src/views/Main/routes.js

```diff
@@ -13,17 +13,20 @@ declare var __LOCAL__;
 
 const auth = new AuthService(Consts.AUTH0_CLIENT_ID, Consts.AUTH0_DOMAIN);
 
+const getBaseUrl = () => {
+  const host = window.location.host;
+  const domainRegex = /(?:[^\.]+\.)*([^\.]+\.[^\.]+)(?:\:\d+)?/;
```

Comment:
I highly recommend writing unit tests to make sure this works as you intend it. for example, it fails in the following case:

```
> 'https://staging.thedataloom.com'.match(domainRegex)
["https://staging.thedataloom.com", "thedataloom.com"]
```

in this example, we want the second element to actually be `staging.thedataloom.com`